### DISCUSSION TITLE
Berechnung mit Bindings

### DISCUSSION THREAD

**Discussion by tschortschde** [Zefau/ioBroker.jarvis#2298] [Q&A]
Hallo,
ich würde gerne zwei Datenpunkte aus dem ioBroker addieren.
Ist das auch mit Bindings möglich?
Kannst Du mir da helfen - ich nutze die pro v.3.1.2

**Comment by mcuiobroker** [comment]
Wie sieht der Geräte-DP aus, wo die Summe angezeigt werden soll?
Sollte so gehen, funktioniert aber nicht? (v3.1.4-beta.6)
`{"default":"(parseInt({val})+parseInt({0_userdata.0.jarvis.test.num2}))"}`

Mit `v3.1.4-beta.6`:
In Anzeigevariante:
`{"default":"Number({0_userdata.0.jarvis.test.num2})+Number({val})"}`

https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/geraet/anzeigevariante/summe-anzeigen

**Comment by tschortschde** [comment]
Vielen Dank!
Ich habe es nun doch mit Blockly und einem separaten eigenen DP umgesetzt.
